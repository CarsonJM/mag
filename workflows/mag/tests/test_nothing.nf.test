nextflow_workflow {

    name "Test workflow: MAG"
    script "../main.nf"
    workflow "MAG"
    tag "workflows"
    tag "mag"
    tag "mag_test_nothing"
    profile "test_nothing"

    test("Parameters: test_nothing") {

        then {
            assertAll(
                { assert workflow.success },
                { assert snapshot(
                    workflow.out
                ).match() }
            )
        }
    }
}
