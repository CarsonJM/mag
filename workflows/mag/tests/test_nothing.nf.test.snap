{
    "Parameters: test_nothing": {
        "content": [
            {
                "0": [
                    [
                        {
                            "id": "test_minigut",
                            "group": "0",
                            "single_end": false
                        },
                        [
                            "test_minigut_1.merged.fastq.gz:md5,b7eaf06531000c4d32a48d050c2df34f",
                            "test_minigut_2.merged.fastq.gz:md5,a1629ec2f2931892a2a0c43f0f677056"
                        ]
                    ],
                    [
                        {
                            "id": "test_minigut_sample2",
                            "group": "0",
                            "single_end": false
                        },
                        [
                            "/nf-core/test-datasets/raw/mag/test_data/test_minigut_sample2_R1.fastq.gz",
                            "/nf-core/test-datasets/raw/mag/test_data/test_minigut_sample2_R2.fastq.gz"
                        ]
                    ]
                ],
                "1": [
                    
                ],
                "10": [
                    "versions.yml:md5,3222fd64337a6c070682b63fc5e0cb3c",
                    "versions.yml:md5,e9b46b38f7df41da088ff41127d3752a"
                ],
                "2": [
                    
                ],
                "3": [
                    
                ],
                "4": [
                    
                ],
                "5": [
                    
                ],
                "6": [
                    
                ],
                "7": [
                    
                ],
                "8": [
                    
                ],
                "9": [
                    
                ],
                "assemblies": [
                    
                ],
                "bin_summary": [
                    
                ],
                "cat": [
                    
                ],
                "genomad": [
                    
                ],
                "metaeuk": [
                    
                ],
                "prodigal": [
                    
                ],
                "prokka": [
                    
                ],
                "refined_bins": [
                    
                ],
                "refined_unbins": [
                    
                ],
                "short_reads": [
                    [
                        {
                            "id": "test_minigut",
                            "group": "0",
                            "single_end": false
                        },
                        [
                            "test_minigut_1.merged.fastq.gz:md5,b7eaf06531000c4d32a48d050c2df34f",
                            "test_minigut_2.merged.fastq.gz:md5,a1629ec2f2931892a2a0c43f0f677056"
                        ]
                    ],
                    [
                        {
                            "id": "test_minigut_sample2",
                            "group": "0",
                            "single_end": false
                        },
                        [
                            "/nf-core/test-datasets/raw/mag/test_data/test_minigut_sample2_R1.fastq.gz",
                            "/nf-core/test-datasets/raw/mag/test_data/test_minigut_sample2_R2.fastq.gz"
                        ]
                    ]
                ],
                "versions": [
                    "versions.yml:md5,3222fd64337a6c070682b63fc5e0cb3c",
                    "versions.yml:md5,e9b46b38f7df41da088ff41127d3752a"
                ]
            }
        ],
        "timestamp": "2024-01-02T16:14:07.274635084"
    }
}